src/httpclient.js

```diff
@@ -9,6 +9,10 @@ const context = process.env.HELIX_FETCH_FORCE_HTTP1
   : fetchAPI;
 const { fetch, AbortError, Headers } = context;
 
+function discard() {
```

Comment:
but in this case, we should always create a new context, otherwise we `reset()` the default context, which might be in use by someone else

@stefan-guggisberg WDYT?

Reply:
d4c24e7 creates a new context in all cases.